src/components/attributes-panel/histogram-tooltip.tsx

```diff
@@ -0,0 +1,44 @@
+// Skip types till this issue in recharts lib will be resolved -
+// Issue - https://github.com/recharts/recharts/issues/2796
+
+// import type { TooltipProps } from "recharts";
+// import type {
+//   ValueType,
+//   NameType,
+// } from "recharts/src/component/DefaultTooltipContent";
+
+import styled from "styled-components";
+
+// type HistogramTooltipProps = TooltipProps<ValueType, NameType> & {
+//   attributeName: string;
+// };
+
+const Container = styled.div`
+  display: flex;
+  padding: 5px;
+  border-radius: 8px;
+  flex-direction: column;
+  background: ${({ theme }) => theme.colors.mainColor};
+`;
+
+const TooltipItem = styled.div`
+  color: ${({ theme }) => theme.colors.fontColor};
+`;
+
+export const HistogramTooltip = ({
+  active,
+  payload,
+  label,
+  attributeName,
+}: any) => {
```

Comment:
add types

Reply:
I added types above but I can't use it because of recharts typing issue. I left comment there.
Issue - https://github.com/recharts/recharts/issues/2796